src/main/java/Event.java

```diff
@@ -0,0 +1,17 @@
+public class Event extends Task{
+    public Event(String description){
+        super(description);
+        this.description = addInDate(description);
+        symbol = "[E]";
+    }
+
+    public String addInDate(String task){
```

Comment:
maybe you could make the date an attribute of the Event class! Might be easier to store and change